frontend/packages/operator-lifecycle-manager/src/components/modals/uninstall-operator-modal.tsx

```diff
@@ -50,6 +80,8 @@ export const UninstallOperatorModal: React.FC<UninstallOperatorModalProps> = ({
                 ).catch(() => Promise.resolve()),
               ]
             : []),
+          canPatchConsoleOperatorConfig &&
```

Comment:
Won't this sometimes pass `false` to `Promise.all`? What will that do? I think we want to fix it so the array only contains promises.

Reply:
[It looks like Promise.all is smart enough to handle it](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all#using_promise.all), but I can change it.